### PR TITLE
OCPBUGS#16610: Updated UPI info in Azure account limits module

### PR DESCRIPTION
**Pull request by dfitzmau**
[OCPBUGS-16610](https://issues.redhat.com/browse/OCPBUGS-16610)

Version(s):
4.14 through to 4.10

Link to docs preview:
* [Azure account limits: content removed](https://63544--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-user-infra#installation-azure-limits_installing-azure-user-infra)
* [Additional Azure configuration parameters](https://63544--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-network-customizations#installation-configuration-parameters_installing-azure-network-customizations)
* [Creating additional worker machines in Azure](https://63544--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-user-infra#installation-creating-azure-control-plane_installing-azure-user-infra)
* [Creating the control plane machines in Azure](https://63544--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-user-infra#installation-creating-azure-control-plane_installing-azure-user-infra)
* [Requirements for using your VNet](https://63544--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-vnet#installation-about-custom-azure-vnet-requirements_installing-azure-vnet)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by dfitzmau** [comment]
rebuild

**Comment by jinyunma** [comment]
Validation get passed when setting zones in installer upi azure arm template, detailed see [comments](https://issues.redhat.com/browse/OCPBUGS-16610?focusedId=22854600&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-22854600).

The update looks good to me.

File: modules/installation-creating-azure-worker.adoc [hunk 1]
```diff
@@ -33,13 +33,14 @@ In this example, you manually launch one instance by using the Azure Resource
...
 additional resources of type `06_workers.json` in the file.
 
+ifdef::azure[]
 [NOTE]
 ====
-If you do not use the provided ARM template to create your worker machines, you
-must review the provided information and manually create the infrastructure. If
-your cluster does not initialize correctly, you might have to contact Red Hat
-support with your installation logs.
+By default, Microsoft {cp} places control plane machines and compute machines in a pre-set availability zone. You can manually set an availability zone for a compute node or control plane node. To do this, modify a vendor's Azure Resource Manager (ARM) template by specifying each of your availability zones in the `zones` parameter of the virtual machine resource.
```
Comment (snarayan-redhat) [inline modules/installation-creating-azure-worker.adoc:39]: I see ARM acronym is already introduced  in Line 33. We can directly mention ARM template. Just a suggestion, up to you.

File: modules/installation-configuration-parameters.adoc [hunk 2]
```diff
@@ -1094,7 +1094,13 @@ ifdef::azure[]
 [id="installation-configuration-parameters-additional-azure_{context}"]
 == Additional Azure configuration parameters
 
-Additional Azure configuration parameters are described in the following table:
+Additional Azure configuration parameters are described in the following table.
+
+[NOTE]
+====
+By default, if you specify availability zones in the `install-config.yaml` file, the installation program distributes control plane and compute machines across link:https://azure.microsoft.com/en-us/global-infrastructure/availability-zones/[these availability zones]
```
Comment (snarayan-redhat) [inline modules/installation-configuration-parameters.adoc:1101]:
```suggestion
By default, if you specify availability zones in the `install-config.yaml` file, the installation program distributes the control plane and the compute machines across link:https://azure.microsoft.com/en-us/global-infrastructure/availability-zones/[these availability zones]
```

**Review by snarayan-redhat — COMMENTED** [review]
Great work! Couple of minor nits and a suggestion.

File: modules/installation-creating-azure-worker.adoc (hunk 1 above)
Comment (dfitzmau) [reply modules/installation-creating-azure-worker.adoc]: Yes, good point. The first instance is a shared paragraph among install docs. Updated.

**Review by r4f4 — COMMENTED** [review]
/lgtm

**Pull request merged**